packages/ruleset/test/schema-description.test.js

```diff
@@ -0,0 +1,284 @@
+const { schemaDescription } = require('../src/rules');
+const { makeCopy, rootDocument, testRule, severityCodes } = require('./utils');
+
+const rule = schemaDescription;
+const ruleId = 'schema-description';
+const expectedSeverity = severityCodes.warning;
+const expectedMsgSchema = 'Schema should have a non-empty description';
+const expectedMsgProp = 'Schema property should have a non-empty description';
+
+describe('Spectral rule: schema-description', () => {
+  describe('Should not yield errors', () => {
+    it('Clean spec', async () => {
+      const results = await testRule(ruleId, rule, rootDocument);
+      expect(results).toHaveLength(0);
+    });
+
+    it('OneOf child schema with no description', async () => {
+      const testDocument = makeCopy(rootDocument);
+
+      testDocument.components.schemas['Soda'].description = undefined;
+
+      const results = await testRule(ruleId, rule, testDocument);
+      expect(results).toHaveLength(0);
+    });
+
+    it('Array items schema with no description', async () => {
+      const testDocument = makeCopy(rootDocument);
+
+      testDocument.components.schemas['MovieNameList'] = {
+        description: 'A list response.',
+        type: 'object',
+        properties: {
+          movieNames: {
+            description: 'List of movie names.',
+            type: 'array',
+            items: {
+              type: 'string'
+            }
+          }
+        }
+      };
+
+      testDocument.paths['/v1/movies'].get.responses['200'].content[
+        'application/json'
+      ].schema = {
+        $ref: '#/components/schemas/MovieNameList'
+      };
+
+      // We should get back a warning ONLY due to the Soda reference in the response (not the oneOf).
```

Comment:
I'm confused by this comment - is it in the right place?

Edit - after reading through the rest of the file, it looks like it might be a copy/paste error